src/Public/Routes.ps1

```diff
@@ -1164,11 +1170,23 @@ function Add-PodeRouteGroup {
         $AllowAnon,
 
         [string[]]
-        $OADefinitionTag
+        $OADefinitionTag,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'file')]
```

Comment:
The `[Parameter()]` block for the `$Routes` parameter for `Add-PodeRouteGroup` will need a `ParameterSetName = 'Script'` property being set as well.

And then at the top of the function, the `[CmdletBinding()]` block will need a `DefaultParameterSetName = 'Script'` property being set.

This way, we allow only `Routes` or `FilePath` to be supplied and not both.

Reply:
added